multiscale_phate/visualize.py

```diff
@@ -102,4 +137,4 @@ def build_condensation_tree(data_pca, diff_op, NxT, merged_list, Ps):
 
         tree = np.vstack((embeddings))
 
-    return tree
+    return tree
```

Comment:
Missing newline. Easiest fix for this is to run `./autoblack.sh` in the root directory to automatically reformat